scipy/optimize/tests/test_cobyla.py

```diff
@@ -1,17 +1,19 @@
 import math
 
 import numpy as np
-from numpy.testing import assert_allclose, assert_, assert_array_equal
-import pytest
+from numpy.testing import assert_allclose, assert_, assert_array_almost_equal
 
-from scipy.optimize import fmin_cobyla, minimize, Bounds
+from scipy.optimize import (fmin_cobyla, minimize, Bounds, NonlinearConstraint,
+    LinearConstraint, OptimizeResult)
```

Comment:
optional nit in case you are pushing other things already

```suggestion
from scipy.optimize import (
    fmin_cobyla, minimize, Bounds, NonlinearConstraint, LinearConstraint,
    OptimizeResult,
)
```